tests/test_models.py

```diff
@@ -94,15 +94,14 @@ def test_add_shoppinglist(self):
     def test_get_shoppinglist(self):
         """Test whether we can get a single shoppinglist"""
         self.test_store.add_user('mash', 'mash@mash2.com', 'mash_pass')
-        self.test_store.add_shoppinglist(3, "hardware")
-        test_item = self.test_store.get_shoppinglist(3, 1)
+        self.test_store.add_shoppinglist(2, "hardware")
+        test_item = self.test_store.get_shoppinglist(2, 1)
         self.assertEquals(
```

Comment:
This is okay and still works. As you move forward you may want to consider using self.assertEqual for comparing numbers mostly and not so much of other data structures. You can using self.assertIn to check if a dictionary key exists in the dictionary.

Reply:
Thank you for the suggestion, makes total sense. I'll start looking into applying this.